go/common/node/node.go

```diff
@@ -71,8 +72,9 @@ type Node struct { // nolint: maligned
 	// Beacon contains information for this node's participation
 	// in the random beacon protocol.
 	//
-	// NOTE: This is reserved for future use.
-	Beacon cbor.RawMessage `json:"beacon,omitempty"`
+	// TODO: This is optional for now, make mandatory once enough
```

Comment:
Should we make this required in master? And optional in the backported version.

Reply:
Depends on how the transition will happen.